frontend/exporter/Cargo.toml

```diff
@@ -13,13 +13,18 @@ description = "Provides mirrors of the algebraic data types used in the Rust com
 rustc_private=true
 
 [dependencies]
-hax-adt-into.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 schemars.workspace = true
-itertools.workspace = true
-hax-frontend-exporter-options.workspace = true
-tracing.workspace = true
+hax-adt-into.workspace = true
 paste = "1.0.11"
-extension-traits = "1.0.1"
-lazy_static = "1.4.0"
+
+tracing = { workspace = true, optional = true }
+itertools = { workspace = true, optional = true }
+extension-traits = { version = "1.0.1", optional = true }
+lazy_static = { version = "1.4.0", optional = true }
+cfg-if = "1.0.0"
+
+[features]
+default = ["full"]
+full = ["dep:tracing", "dep:itertools", "dep:extension-traits", "dep:lazy_static"]
```

Comment:
What exactly is this feature? Full isn't a great name. Something a little more descriptive would be nice. Adding some comments here to describe what it enables would be good too.